aldryn_newsblog/search_indexes.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.template import RequestContext
+
+from aldryn_search.utils import get_index_base, strip_tags
+
+from aldryn_newsblog.models import Article
+
+
+class ArticleIndex(get_index_base()):
+    haystack_use_for_indexing = getattr(settings, "ALDRYN_NEWSBLOG_SEARCH", True)
+
+    INDEX_TITLE = True
+
+    def get_title(self, obj):
+        return obj.safe_translation_getter('title')
+
+    def get_description(self, obj):
+        return obj.safe_translation_getter('lead_in')
+
+    def get_index_kwargs(self, language):
```

Comment:
It's needed to comply with parler language fallback mechanism, langage code must be passed to the ``filter`` method to access the translatable fields.